functions/src/modules/SQR.functions.ts

```diff
@@ -438,50 +395,40 @@ export const syncAllotments = functions.database
     ): Promise<any> => {
       const { listName, fileName } = context.params;
       const changedValues = change.after.val();
-      const spreadsheetId = functions.config().sqr.spreadsheet_id;
 
-      const gsheets = new GoogleSheet();
-      const allotmentFileNames = await gsheets.getColumn(
-        spreadsheetId,
-        ISoundQualityReportSheet.Allotments,
-        'File Name'
+      const gsheets = new GoogleSheets(
+        functions.config().sqr.spreadsheet_id,
+        ISoundQualityReportSheet.Allotments
       );
-
+      const allotmentFileNames = await gsheets.getColumn('File Name');
       const rowNumber = allotmentFileNames.indexOf(fileName) + 1;
       const { languages, notes, soundQualityReporting } = changedValues;
+
       const {
         timestampGiven,
         assignee,
         timestampDone,
         followUp,
       } = soundQualityReporting;
-      const updateResults = await gsheets.updateAllotmentRow(
-        spreadsheetId,
-        ISoundQualityReportSheet.Allotments,
-        rowNumber,
-        {
-          days_passed: EMPTY_VALUE,
-          date_given: spreadsheetDateFormat(timestampGiven),
-          notes: withDefault(notes),
-          language: commaSeparated(languages),
-          status: soundQualityReporting.status,
-          file_name: fileName,
-          devotee: withDefault(assignee.name),
-          email: withDefault(assignee.emailAddress),
-          phone: EMPTY_VALUE,
-          location: EMPTY_VALUE,
-          date_done: spreadsheetDateFormat(timestampDone),
-          follow_up: withDefault(followUp),
-          list: withDefault(listName),
-          serial: EMPTY_VALUE,
-        }
-      );
+
+      const updateResults = await gsheets.updateRow(rowNumber, {
+        "Days passed": spreadsheetDateFormat(timestampGiven),
+        "Date Given": withDefault(notes),
+        "Language": commaSeparated(languages),
+        "Status": soundQualityReporting.status,
+        "File Name": fileName,
+        "Devotee": withDefault(assignee.name),
+        "Email": withDefault(assignee.emailAddress),
+        "Date Done": spreadsheetDateFormat(timestampDone),
+        "Follow Up": withDefault(followUp),
+        "List": withDefault(listName),
```

Comment:
Don't update list!

Reply:
Alright, then I need to pull the existing row.